### PR TITLE
#914 fixes some typos.

### LINKED ISSUE #914: MULIT typos
**Describe the bug**
There are various typos for mulit/multi across the system

**Janeway version**
All

**To Reproduce**
 https://github.com/BirkbeckCTP/janeway/search?q=mulit&unscoped_q=mulit

**Expected behavior**
Should be spelled correctly

### PR DESCRIPTION
**Pull request by ajrbyers**
Closes #914

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -90,6 +90,7 @@ src/media/profile_images/*
 src/urls\.txt
 
 src/files/styling/
+src/files/bepress/
```
Comment (mauromsl) [inline .gitignore]: Could use a comment here as this is out of context without the plugin

Comment (ajrbyers) [reply .gitignore]: Maybe we should have a folder for plugins to store their files in and ignore that?

Comment (mauromsl) [reply .gitignore]: Sounds good to me, `src/files/plugins/<plugin_name>` ?

Comment (ajrbyers) [reply .gitignore]: Exactamundo! I'll make the change to this and the bepress plugin.

**Comment by ajrbyers** [comment]
Change made: https://github.com/BirkbeckCTP/bepress/commit/7b50b56aef5284d7dc9f03269a9ba784e3a6de71

**Pull request merged**